apps/website/src/content/docs/tree.mdx

```diff
@@ -15,6 +15,14 @@ thumbnail: Tree
 
 A tree can be used to organize data in an application specific way, or it can be used to sort, filter, group, or search data as the user deems appropriate. Each line of data level may begin with an eye icon for toggling its visibility. In this context, this icon is called specialty checkbox.
 
+## Size
+
+There are 2 different sizes available, which can be applied to any tree. The medium size is a default and should always be the first choice, unless there are good reasons to use the small version.
```

Comment:
I don't believe we use the term "medium" anywhere. There's a "default" size and a "small" size. Also we should mention the `size` prop.

```suggestion
There are 2 different sizes available. The default size should suffice for most cases. When a smaller version of the tree is needed, use `size="small"`.
```

(Rephrase as necessary)

Reply:
Ok, I'll go with your suggestion.  Just a heads up though that I copy / pasted that text from [the buttons page](https://itwinui.bentley.com/docs/button#size), so we should also remove medium from there as well.